battles/helpers/battle.py

```diff
@@ -0,0 +1,78 @@
+from collections import Counter
+
+from battles.helpers.emails import send_email_when_battle_finishes
+from battles.helpers.fight import compare_two_pokemons
+from battles.models import BattleTeam
+from users.models import User
+
+
+def can_run_battle(battle):
+    try:
+        BattleTeam.objects.get(
+            battle_related=battle, trainer=battle.creator)
+        BattleTeam.objects.get(
+            battle_related=battle, trainer=battle.opponent)
+    except BattleTeam.DoesNotExist:
+        return False
+    else:
+        return True
+
+
+def mount_battle_list(battle):
+    creator_team = BattleTeam.objects.get(
+        battle_related=battle, trainer=battle.creator)
+    opponent_team = BattleTeam.objects.get(
+        battle_related=battle, trainer=battle.opponent)
+    result = {}
+    result['creator_team'] = [
+        pokemon.id for pokemon in creator_team.pokemons.all()]
+    result['opponent_team'] = [
+        pokemon.id for pokemon in opponent_team.pokemons.all()]
+    return result['creator_team'], result['opponent_team']
+
+
+def get_winner_pokemon_list(battle):
+    battle_list = mount_battle_list(battle)
+    comparison_winners = []
+    for creator_pokemon, opponent_pokemon in zip(battle_list[0], battle_list[1]):
+        comparison_winners.append(compare_two_pokemons(
+            creator_pokemon, opponent_pokemon))
+    return comparison_winners
+
+
+def get_the_battle_winner(battle):
```

Comment:
In general, it's better to avoid putting articles ('the') in method names, unless you actually need to explicit something :)